view/frontend/web/js/certificates-sdk.js

```diff
@@ -0,0 +1,19 @@
+define(['sdkToken'], function (sdkToken) {
+    return function (container, params) {
+        return new Promise(function (resolve, reject) {
```

Comment:
IE 11 doesn't support Promises: https://caniuse.com/#search=Promise M2 doesn't have a polyfill for this, does it? If it doesn't, please refactor.